external_server_main.py

```diff
@@ -1,66 +1,77 @@
 #!/usr/bin/env python3
-import logging
-import logging.handlers
 import sys
+import argparse
 import os
+import json
 
-from rich.logging import RichHandler
+from external_server.server import ExternalServer, eslogger as eslogger
+from external_server.config import load_config, InvalidConfiguration
+from external_server.logs import configure_logging
 
-dir_path = os.path.dirname(os.path.abspath(__file__))
-sys.path.append(os.path.join(dir_path, "./lib/fleet-protocol/protobuf/compiled/python"))
 
-from external_server.utils import argparse_init
-from external_server.external_server import ExternalServer
-from external_server.config import load_config, InvalidConfigError
-from external_server import constants
+def parsed_script_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-c",
+        "--config",
+        type=str,
+        default="./config/config.json",
+        help="path to the configuration file",
+    )
+    parser.add_argument("--tls", action=argparse.BooleanOptionalAction, help="tls authentication")
+    tls = parser.add_argument_group("tls", description="if tls is used, set following arguments")
+    tls.add_argument("--ca", type=str, help="path to Certificate Authority certificate files")
+    tls.add_argument("--cert", type=str, help="path to PEM encoded client certificate file")
+    tls.add_argument("--key", type=str, help="path to PEM encoded client private keys file")
+
+    args = parser.parse_args()
+
+    if not os.path.isfile(args.config):
+        raise FileNotFoundError(f"Config file {os.path.abspath(args.config)} not found.")
+    if args.tls:
+        missing_fields = []
+        if not args.ca:
+            missing_fields.append("ca certificate")
```

Comment:
```suggestion
            missing_fields.append("ca certificate (--ca)")
```
do this for all missing_fields.